packages/fast-components/src/anchored-region/anchored-region.ts

```diff
@@ -0,0 +1,978 @@
+import { attr, FastElement, observable, ref } from "@microsoft/fast-element";
+import { isNil } from "lodash-es";
+import rafThrottle from "raf-throttle";
```

Comment:
I don't think raf-throttle is an enumerated dependency. Neither is lodash-es. This is likely only working because of Yarn workspaces resolving to the root node_modules dir.

I think we can avoid taking the dep on raf-throttle though and hook into `fast-element`'s `DOM.queueUpdate` method

Reply:
I think queueUpdate addresses a different concern.  In any case I took out the throttles for now as the functions invoked by resize merely store values and queue an update so while we could get a lot of updating of values we don't trigger excess renders.  We will need some generic throttle functionality tho.

Reply:
Basically, queueUpdate manages a single queue of tasks to run on the next animation frame. When you queue something, if the raf isn't scheduled, it will schedule one and then add your task to the queue. If it's already schedule, it just adds your task to the current queue. It's good to use our queueUpdate method because it ensures all our work is scheduled in the same way.

If you need to throttle the rate at which an individual component schedules things in tehe queue, that would need to be handled on a per-component basis. Individual bindings do this, so they don't schedule more than one update per frame.

On a related note, what 3rd party libraries are we using in our web components. This is the first that I've noticed a dependency outside of our own. Are we using lodash elsewhere or any other library? I'd like to avoid deps if we can.